draft-ietf-tls-rfc4492bis.xml

```diff
@@ -615,53 +608,52 @@
       </section>
       <section anchor="cke" title="Client Key Exchange">
         <t> When this message is sent:</t>
-        <t> This message is sent in all key exchange algorithms.  If client authentication with ECDSA_fixed_ECDH
-          or RSA_fixed_ECDH is used, this message is empty.  Otherwise, it contains the client's ephemeral ECDH
+        <t> This message is sent in all key exchange algorithms.  It contains the client's ephemeral ECDH
           public key.</t>
         <t> Meaning of the message:</t>
         <t> This message is used to convey ephemeral data relating to the key exchange belonging to the client (such
           as its ephemeral ECDH public key).</t>
         <t> Structure of this message:</t>
-        <t> The TLS ClientKeyExchange message is extended as follows.<figure><artwork><![CDATA[
+        <t> The TLS ClientKeyExchange message is extended as follows.</t>
+        <t><figure><artwork><![CDATA[
         enum {
             implicit,
             explicit
         } PublicValueEncoding;
-]]></artwork></figure><list style="symbols">
+]]></artwork></figure></t>
+        <t><list style="symbols">
           <t> implicit, explicit: For ECC cipher suites, this indicates whether the client's ECDH public
             key is in the client's certificate ("implicit") or is provided, as an ephemeral ECDH public key, in the
-            ClientKeyExchange message ("explicit").  (This is "explicit" in ECC cipher suites except when the client
-            uses the ECDSA_fixed_ECDH or RSA_fixed_ECDH client authentication mechanism.)<figure><artwork><![CDATA[
+            ClientKeyExchange message ("explicit").  The implicit encoding is deprecated and is retained here for 
+            backward compatibility only.</t></list></t>
```

Comment:
Like this?

struct {
                ECPoint ecdh_Yc;
        } ClientECDiffieHellmanPublic;